scripts/reports/count-non-stricken-event-codes-by-year.ts

```diff
@@ -0,0 +1,134 @@
+// usage: npx ts-node --transpile-only scripts/reports/count-non-stricken-event-codes-by-year.ts M071,M074 2021-2022 [includeStricken]  > ~/Desktop/m071s-and-m074s-filed-2021-2022.csv
+
+import { count } from '@web-api/persistence/elasticsearch/searchClient';
+import { createApplicationContext } from '@web-api/applicationContext';
+import { requireEnvVars } from '../../shared/admin-tools/util';
+import { validateDateAndCreateISO } from '@shared/business/utilities/DateHandler';
+
+requireEnvVars(['ENV', 'REGION']);
+
+const includeStricken = !!(process.argv.length > 4);
+if (includeStricken) {
+  console.log('including stricken docket entries');
+}
```

Comment:
Is this still necessary? I thought the argument for not being verbose in the ultimate output was so this report could be used in other scripts.